openerp/tools/translate.py

```diff
@@ -366,7 +366,8 @@ def next(self):
                 return self.next()
 
             if not line.startswith('msgid'):
-                raise Exception("malformed file: bad line: %s" % line)
+                _logger.info(_('Malformed file: Bad line: %s') % (line), exc_info=True)
+                raise openerp.exceptions.UserError(_('Malformed file \n\n Bad line: %s') % (line))
             source = unquote(line[6:])
```

Comment:
* do not use translation for logger info.
* there is no need of exc_info args, here no exception raise so logger don't log any trackback.
* from openerp.exceptions import UserError